copagame.iml

```diff
@@ -39,7 +39,7 @@
     <orderEntry type="library" name="Deps: org.clojure/tools.analyzer.jvm:0.7.2" level="project" />
     <orderEntry type="library" name="Deps: io.pedestal/pedestal.log:0.5.9" level="project" />
     <orderEntry type="library" name="Deps: lambdaisland/deep-diff:0.0-47" level="project" />
-    <orderEntry type="library" name="Deps: com.fasterxml.jackson.dataformat/jackson-dataformat-cbor:2.9.6" level="project" />
+    <orderEntry type="library" name="Deps: com.fasterxml.jackson.dataformat/jackson-dataformat-cbor:2.12.4" level="project" />
```

Comment:
This file seems to be an intelij file? Should we add it to `.gitignore` and remove from the source too?

Reply:
Nice! ✂️ and add to .gitignore